### PR TITLE
[Teleop] Control improvements

### PR DESCRIPTION
**Pull request by CameronTressler**
Add useful IK buttons to science GUI
Update science presets
Add wrist turn count functionality
Attempt to lock joints that don't move in motion planning

### CODE REVIEW

File: base_station/gui/src/components/ArmControls.vue [hunk 1]
```diff
@@ -6,6 +6,9 @@
     <div class="keyboard">
       <GimbalControls/>
     </div>
+    <div style="font-color: abs(wristTurnCount) >= 2 ? red : black;">
```
Comment (CameronTressler) [inline base_station/gui/src/components/ArmControls.vue:9]: @catkims this doesn't seem to work properly, not sure the syntax to fix

Comment (CameronTressler) [reply base_station/gui/src/components/ArmControls.vue:9]: This shouldn't block a merge though

**Review by tabiosg — APPROVED** [review]
Just to make it clear, wrist turn count hasn't been implemented yet on esw's end, hopefully that doesn't break anything

**Comment by CameronTressler** [comment]
> Just to make it clear, wrist turn count hasn't been implemented yet on esw's end, hopefully that doesn't break anything

It shouldn't. Especially not for science arm tomorrow

File: base_station/gui/src/components/ArmControls.vue (hunk 1 above)
Comment (catkims) [reply base_station/gui/src/components/ArmControls.vue:9]: I'm looking into it now

**Pull request merged**